### ISSUE TITLE
How to set up the start position of the scrolbar?

### ISSUE THREAD

**Issue by mario-lemes** [smikhalevski/react-scroll-box#4]
Hey again @smikhalevski,

Is there any way to set up the start position of the scrollbar on the rendering? I mean, instead of render in the top part let say in the bottom part. Or even specify a position in pixels. I saw there are two variables involved on the position (targetX and targetY), would be possible to set up as props?

Thank for all your proactiveness and help!

**Comment by smikhalevski** [comment]
Hello @Yonirt,

You are welcome!

Use [`scrollTo(x, y, 0)`](https://github.com/smikhalevski/react-scroll-box#void-scrollto-x-y-duration--0-silent--false) in `componentDidMount`. Here is an example:

``` jsx
class MyComponent extends React.Component {
  componentDidMount() {
    this.refs.scrollBox.scrollTo(100, 100, 0); // This would scroll to x=100, y=100 with no animation
  }
  render() {
    return <ScrollBox ref="scrollBox">FooBar</ScrollBox>;
  }
}
```

**Comment by mario-lemes** [comment]
Hello @smikhalevski,

Actually I did that but before scrolling to the position I want with scrolTo() function, the "onViewPortScroll" is triggering on (0,0). I assume because is the position where is rendering by the component. My question is, if it would be possible to render the component directly on (100,100) for instance.

The reason is because I want to trigger a function when the scrollbar reach the top but not in the moment is rendered, just when is scrolled by the user.

**Comment by smikhalevski** [comment]
Oh, I see. This  an issue because `onViewportScroll` should not fire before any actual scrolling occurs.  The snippet I provided is the recommended way to initialise scrolling position. As a temporary workaround I would recommend to ignore the first time when `onViewportScroll` triggers. I'm going to investigate this issue and return back to you.

**Comment by smikhalevski** [comment]
Fixed in 1594ae06076b0909ee1588e74b80f9cb8cc0eee7
Published v0.1.3
